tests/unittests/Makefile

```diff
@@ -215,6 +215,7 @@ ifneq (, $(filter $(AVR_BOARDS), $(BOARD)))
 endif
 
 LARGE_STACK_TESTS += tests-hacl
+LARGE_STACK_TESTS += tests-libcose
```

Comment:
What's the difference of putting them all in one line? I think this looks rather hacky and only for "good reading" purposes. So I'd say this variable, since is repeatedly needed, should be declared somewhere else and being set in the correspondent package. Actually how it was before it wasn't bad, but it depends on what's the purpose of the formatting.

Reply:
The lines are only split for reading purposes. If you want them declared in the separate packages I suggest we do that in a follow up as it would need multiple line changes such as moving L220-L222 below L227 and that is going outside the scope of this PR.

Reply:
OK.